src/python/pants/backend/python/subsystems/lambdex.py

```diff
@@ -50,6 +50,11 @@ class Lambdex(PythonToolBase):
             as recommended by cloud vendors.  (If you are using `python_awslambda`, you will need to
             also update the handlers configured in the cloud from `lambdex_handler.handler` to
             `lambda_function.handler`.)
+
+            See the docs for more details:
+
+            * https://www.pantsbuild.org/v2.18/docs/awslambda-python#migrating-from-pants-216-and-earlier
```

Comment:
When cherry-picking I'll update these URLs to `v2.17`.

I'm also not sure if this is the right slug (that is, ReadME will remove the `.` from `2.16`), but I don't know how to confirm. I can confirm/update live once deployed for the first time.

Reply:
We use the `doc_url()` function for this: https://github.com/pantsbuild/pants/blob/cae2e9ae7540cd1932ebe24a23b74356532e15d2/src/python/pants/util/docutil.py#L19